reference-app/src/main/java/org/smartregister/anc/application/AncApplication.java

```diff
@@ -59,7 +59,6 @@ public void onCreate() {
         CoreLibrary.init(context, new AncSyncConfiguration(), BuildConfig.BUILD_TIMESTAMP);
         AncLibrary.init(context, BuildConfig.DATABASE_VERSION, new ANCEventBusIndex());
         ConfigurableViewsLibrary.init(context);
-        setDefaultLanguage();
```

Comment:
Does this mean even in the initial setup the app uses the phone's default language?

Reply:
@junaidwarsivd  any feedback?